### ISSUE TITLE
Docs are missing new script injection tools

### ISSUE THREAD

**Issue by fnimick** [svecosystem/mode-watcher#101]
### Change Type

Addition

### Proposed Changes

Should ideally have an example of how to do this in `hooks.server.ts`, and add docs to the new `generateSetInitialModeExpression` function.

I can take a stab at this later if you'd like.

**Comment by huntabyte** [comment]
Why would it require manual changes to `app.html`? We should be able to add the appropriate class/theme via the `html` prop on the `transformPageChunk` no?

**Comment by fnimick** [comment]
I don't think that works, since we need to inject javascript to run to read the client's local storage and set the class correctly, no? We don't have access to this at initial render time.

My solution is to add the following to app.html:
```
    <script nonce="%sveltekit.nonce%">
      %modewatcher.snippet%
    </script>
```

And the following server-side hook:
```
const injectHeadScripts: Handle = async ({ event, resolve }) => {
  return resolve(event, {
    transformPageChunk: ({ html }) => {
      return html.replace("%modewatcher.snippet%", generateSetInitialModeExpression({}));
    },
  });
};
```

**Comment by huntabyte** [comment]
We'd use a cookie for the server side instead.

**Comment by huntabyte** [comment]
@fnimick does the solution you proposed work for you consistently? As in you don't have FOUC or anything of that sort?

**Comment by jprinaldi** [comment]
@huntabyte I've been using this solution for several projects and so far it's been working flawlessly.

**Comment by huntabyte** [comment]
Awesome, will add it to the docs then!
